### PR TITLE
[security] Set owners of codeowners file

### PR DESCRIPTION
**Pull request by hspinks**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [ ] ~Tested your changes, especially after any code review iterations.~
- [ ] ~Included any relevant screenshots of UI updates.~
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

## What does this PR accomplish?

Sets the owners of the CODEOWNERS file

## Did you add any dependencies?

N/A

## How did you test the change?

N/A

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,2 @@
+.github/CODEOWNERS @SamMousa @brunobowden @advayDev1 @hspinks
```
Comment (advayDev1) [inline .github/CODEOWNERS]:
```suggestion
/.github/CODEOWNERS @SamMousa @brunobowden @advayDev1 @hspinks @crazybob
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,2 @@
...
 .github/workflows @SamMousa @brunobowden
```
Comment (advayDev1) [inline .github/CODEOWNERS]:
```suggestion
/.github/workflows @SamMousa @brunobowden
```

Technically the `/` should be there

**Comment by advayDev1** [comment]
@brunobowden we think this is a good balance, let us know what you think.

**Comment by hspinks** [comment]
Closing in favor of #503

**Pull request closed without merging**